### PR TITLE
Migrate the OSG PKI command line client documentation.

### PR DESCRIPTION
**Pull request by bbockelm**
Original URL: https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/OSGPKICommandlineClients

This document was in very poor shape.  It still isn't very good, but we can save a rewrite for a different day.

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

File: docs/common/pki-cli.md [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+
+### osg-gridadmin-cert-request
+
+Request and retrieve multliple host certificates from OIM. Authenticates to OIM and is only for use by Grid Admins for certificates they are authorized to approve. This script is only supported with all hosts being in the same domain (so we ensure they go to the same Grid Admin). The certificates are stored with the format of `hostname-requestid.pem` (i.e. the id generated from the request for the certificate). The key is stored as `hostname-serial-key.pem`.
+
+This script does the following in the process of acquiring certificates for the hostnames specified:
+
+-   Reads a list of fully-qualified hostnames from a file specified by the user.
+-   For each hostname:
+    -   Generates a new private key and CS`.R
```
Comment (matyasselmeci) [inline docs/common/pki-cli.md]: Typo here (`` `.R``)

File: docs/common/pki-cli.md [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+```
+
+Test Mode
+---------
+
+The scripts have two modes of execution.
+
+In the normal mode of execution, the script connects to the production server and generated certificates are from default OSG CA.
+
+If the user provides a '-T' parameter on the command-line, the scripts connect to the OIM-ITB server and any generated certificates are issued by the OSG test CAs. This mode is intended for testing and training. The resulting certificates are not usable in a production environment.
```
Comment (matyasselmeci) [inline docs/common/pki-cli.md]: `'-T'` should be `` `-T` ``

**Comment by bbockelm** [comment]
@matyasselmeci - ready for re-review.

File: docs/common/pki-cli.md [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+
+### osg-gridadmin-cert-request
+
+Request and retrieve multliple host certificates from OIM. Authenticates to OIM and is only for use by Grid Admins for certificates they are authorized to approve. This script is only supported with all hosts being in the same domain (so we ensure they go to the same Grid Admin). The certificates are stored with the format of `hostname-requestid.pem` (i.e. the id generated from the request for the certificate). The key is stored as `hostname-serial-key.pem`.
+
+This script does the following in the process of acquiring certificates for the hostnames specified:
+
+-   Reads a list of fully-qualified hostnames from a file specified by the user.
+-   For each hostname:
+    -   Generates a new private key and `CSR`.
```
Comment (matyasselmeci) [inline docs/common/pki-cli.md]: That's not what I meant. "CSR" shouldn't be monospace here at all since it's not monospace anywhere else.

Comment (bbockelm) [inline docs/common/pki-cli.md]: Roger.  FWIW - for typos like this, feel free to edit directly instead of commenting and waiting for the round trip.

**Pull request merged**